native/src/components/MalteHelpFormOffer.tsx

```diff
@@ -114,56 +114,59 @@ const MalteHelpFormOffer = ({
 
   return (
     <Container>
-      <InformationRow>
-        <Icon Icon={SupportIcon} />
-        <InformationText>{t('supportNote')}</InformationText>
-      </InformationRow>
-      <InformationRow>
-        <Icon Icon={SecurityIcon} />
-        <InformationText>{t('securityNote')}</InformationText>
-      </InformationRow>
-
-      <FormInput name='name' title={t('name')} control={control} rules={{ required: true }} />
-      <FormInput name='roomNumber' title={t('roomNumber')} control={control} showOptional />
-
-      <View>
-        <InputTitle>{t('howToBeContacted')}</InputTitle>
-        <FormRadioButtons
-          name='contactChannel'
+      {/* @ts-expect-error Android avoids the keyboard automatically */}
+      <KeyboardAvoidingView behavior={Platform.OS === 'ios' ? 'padding' : ''}>
```

Comment:
:x: Please set it to `undefined` instead of `''` for android to avoid the error.
```suggestion
      <KeyboardAvoidingView behavior={Platform.OS === 'ios' ? 'padding' : undefined}>
```